docs/src/cli/distribution_strategies.md

```diff
@@ -0,0 +1,40 @@
+# Distribution Strategies of the CLI
+
+The CLI offers several distribution strategies to execute tasks, each designed for different execution environments. You can choose one of the following:
+
+## SLURM
+
+Deploy tasks on a SLURM cluster to leverage distributed computing resources. When using the `slurm` strategy, you can configure SLURM-specific options to fine-tune your job scheduling and resource allocation. For example, you may supply job resources via parameters like:
+
+```bash
+--job-resources='{"mem": "10M", "cpus_per_task": "2", "ntasks": "4", "partition": "debug", "time": "00:05:00"}'
```

Comment:
```suggestion
--job-resources='{"mem": "10M", "time": "00:05:00", "partition": "cpu"}'
```
Most clusters require mem and time; and sometimes partition. Other settings are rare.